data-otservbr-global/scripts/talkactions/gm/push_creature.lua

```diff
@@ -1,10 +1,13 @@
 local pushCreature = TalkAction("/c")
 
 function pushCreature.onSay(player, words, param)
-	if not player:getGroup():getAccess() or player:getAccountType() < ACCOUNT_TYPE_GOD then
+	if not player:getGroup():getAccess() or player:getAccountType() < ACCOUNT_TYPE_GAMEMASTER then
```

Comment:
I'm not super sure about this one, but I feel like GMs could abuse this. Not every GM on big servers is 100% trusted, their role is limited, so being able to pull players seems strong.

```suggestion
	if not player:getGroup():getAccess() or player:getAccountType() < ACCOUNT_TYPE_GOD then
```